packages/cli/api-importers/openapi/openapi-ir-to-fern/src/buildTypeDeclaration.ts

```diff
@@ -610,3 +652,7 @@ function convertPropertyTypeReferenceToTypeDefinition(
         };
     }
 }
+
+function getInline(declarationDepth: number): boolean | undefined {
+    return declarationDepth > 0 ? true : undefined;
+}
```

Comment:
It might be worth leaving a function-level comment on this one - I could see us forgetting the rationale around how we're detecting whether or not a type is in-lined. It looks like this was inspired by [this comment](https://github.com/fern-api/fern/pull/5248#discussion_r1852938762)?